org.adempiere.base/src/org/compiere/model/MUser.java

```diff
@@ -194,16 +194,21 @@ public static MUser getCopy(Properties ctx, int AD_User_ID, String trxName)
 		return user;
 	}
 	
+	public static MUser get (Properties ctx, String name, String password)
+	{
+		return MUser.get(ctx, name, password, false);
+	}
+	
 	/**
 	 * 	Get User
 	 *	@param ctx context
 	 *	@param name name
 	 *	@param password password
```

Comment:
should change to `password is ignore when isSSOLogin is true`

Reply:
Done